flixel/addons/tile/FlxTilemapExt.hx

```diff
@@ -539,6 +524,79 @@ class FlxTilemapExt extends FlxTilemap
 	}
 	
 	/**
+	 * Sets the slope arrays, which define which tiles are treated as slopes of 22.5 degrees.
+	 * 
+	 * @param 	HighSlopes 	An array containing the numbers of the tiles to be treated as high slope tiles with a slope on the left.
+	 * @param 	LowSlopes	An array containing the numbers of the tiles to be treated as low slope tiles with a slope on the right.
+	 */
+	public function setSlopes22(?HighSlopes:Array<Int>, ?LowSlopes:Array<Int>) 
```

Comment:
It's your decision, guys..
`setSteep(HeelSlopes, ToeSlopes)` / `setGentle(HeelSlopes, ToeSlopes)` ?
I have to do something more here?

And I have another modified class, a FlxWaveSprite with vertical mode that fix https://github.com/HaxeFlixel/flixel-addons/issues/113
To pull-request it, do I have to make another fork? or another branch? or just pull-request together here?